Documentation/gettingstarted/hubble.rst

```diff
@@ -40,7 +40,7 @@ Enable the Hubble UI by running the following command:
             cilium hubble enable --ui
             🔑 Found existing CA in secret cilium-ca
             ✨ Patching ConfigMap cilium-config to enable Hubble...
-            ♻️  Restarted Cilium pods
+            ♻️ Restarted Cilium pods
```

Comment:
Is this one simply a typo in the docs, or does Hubble mistakenly add this additional space? Would be nice to fix Hubble as well, if it's responsible for the typo.